phpstan.neon.dist

```diff
@@ -2,13 +2,12 @@ includes:
     - phpstan-baseline.neon
 
 parameters:
-    level: 5
-
+    level: max
```

Comment:
we should stop using max, on phpstan 1.0 max is level 9, currently is level 8

Reply:
Not a big deal since it require the bump from ^0.12 to ^1.0

Reply:
Fine, it is either changing it now or later, most likely we are not going to pass level 9 that easy